Autocomplete/recent-searches/src/index.js

```diff
@@ -0,0 +1,39 @@
+import algoliasearch from 'algoliasearch/lite';
+import { autocomplete, highlightHit } from '@algolia/autocomplete-js';
+import { createLocalStorageRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+
+const recentSearches = createLocalStorageRecentSearchesPlugin({
+  key: 'RECENT_SEARCH',
+  limit: 5,
+});
+
+const searchClient = algoliasearch(
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
+);
+const index = searchClient.initIndex('instant_search');
+
+autocomplete({
+  container: '#autocomplete',
+  plugins: [recentSearches],
+  openOnFocus: true,
+  onSubmit({ state }) {
+    index.search(state.query).then((result) => {
```

Comment:
I feel like the results should also be shown on any input, typing in an input and it doing nothing was very confusing for me at first. Maybe you can put it in a function like `searchAndShowMainResults` to make it easier to replace the function call with an InstantSearch refine

Reply:
Non search-as-you-type was kind of confusing to me. You can try the [preview](https://codesandbox.io/s/github/algolia/doc-code-samples/blob/feat/plugins/Autocomplete/recent-searches) again. How is it now?

Reply:
maybe more clean like this with a note? https://codesandbox.io/s/lucid-torvalds-gwf3m?file=/index.html

Reply:
https://github.com/algolia/doc-code-samples/pull/232/commits/f2464cc0ef0ea99fa2b5dbd76afe22d0d34ed619